challenge_receptionist/src/challenge_receptionist/receptionist.py

```diff
@@ -0,0 +1,148 @@
+import rospy
+import smach
+import datetime
+import robot_smach_states as states
+import robot_smach_states.util.designators as ds
+
+from robocup_knowledge import load_knowledge
+from robot_skills.util.entity import Entity
+from hmi_msgs.msg import QueryResult
+
+challenge_knowledge = load_knowledge('challenge_receptionist')
+
+
+class ChallengeReceptionist(smach.StateMachine):
```

Comment:
Breaking this up in multiple sub-state-machines allows for easier re-use and easier testing